### PR TITLE
Font size correction in table

### LINKED ISSUE #6986: Threat Hunting dashboard - Mismatch between alert fonts in the events section
# Description 

There's a mismatch between alert fonts (Compliance fields) in the events section of the Threat Hunting Dashboard.

# Information 

When displaying fields that are part of Wazuh alerts on the Threat Hunting dashboard, we can observe that not all of them share the same size.

We need to normalize the size of the details fonts.

fontsizing

### PR DESCRIPTION
**Pull request by chantal-kelm**
### Description

Font size correction in table

### Issues Resolved

#6986 

### Evidence

image

image

image

image

image

image

image

image

image

image

image

image

image

image

### Test

- Verify in the Threat Hunting dashboard table that the font size for all fields and their displayed results is 13px.

- Verify in the events tab table of each module that the font size for all fields and their displayed results is 13px.

- Verify in the table of the inventory tab of the Vulnerabilities module that the font size for all fields and their displayed results is 13px.

- Test it with different `rule.gdpr` values (single value, multiple gdpr's, no value).

image

### Check List
- [ ] All tests pass
  - [ ] `yarn test:jest`
- [ ] New functionality includes testing.
- [ ] New functionality has been documented.
- [ ] Update [CHANGELOG.md](./../CHANGELOG.md)
- [ ] Commits are signed per the DCO using --signoff

### CODE REVIEW

**Review by Tostti — DISMISSED** [review]
Test 🟢 
image
image

CR 🟢 

LGTM

**Comment by guidomodarelli** [comment]
CR
Test :green_circle: 

image
image
image

**Review by guidomodarelli — DISMISSED** [review]

File: plugins/main/public/components/common/wazuh-discover/render-columns.tsx [hunk 1]
```diff
@@ -13,7 +13,7 @@ export const MAX_ENTRIES_PER_QUERY = 10000;
 
 const renderRequirementsSecurityOperations = (value: []) => {
   return (
-    <EuiText gutterSize='s' direction='column'>
+    <EuiText style={{ fontSize: '13px' }} gutterSize='s' direction='column'>
```
Comment (Desvelao) [inline plugins/main/public/components/common/wazuh-discover/render-columns.tsx:16]: **nitpick**: replace the usage of `EuiText` by returning the raw text joined by comma. We should ensure the value is an array before using the `.join` method.

**Comment by guidomodarelli** [comment]
CR 
TEST

image
image

**Review by guidomodarelli — APPROVED** [review]

**Review by asteriscos — APPROVED** [review]

**Pull request merged**